README.md

```diff
@@ -102,6 +102,12 @@ and run:
 npm install
 ```
 
+* Note: package-lock.json file was generated by Node version 8 and npm version 6.4.1. Although the app supports Node versions 6 and 8 as well as multiple versions of npm, we should always use those versions when updating package-lock.json, * unless it is decided to upgrade those, and in this case the readme should be updated as well. If using a different version for node OR npm, please delete the package-lock.json file prior to installing node packages and also prior to pushing to github.
```

Comment:
Text makes much more sense now. Just one final formatting note. I would remove the list here since this is not really a list, and also remove the `*` before `unless it is decided`.